tests/Unit.Tests/Controllers/CourseController.Tests.cs

```diff
@@ -0,0 +1,81 @@
+
+
+using System;
+using System.Collections.Generic;
+using GovUk.Education.SearchAndCompare.Domain.Client;
+using GovUk.Education.SearchAndCompare.Domain.Models;
+using GovUk.Education.SearchAndCompare.UI.Controllers;
+using GovUk.Education.SearchAndCompare.UI.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+
+namespace GovUk.Education.SearchAndCompare.UI.Unit.Tests.Controllers
+{
+    [TestFixture]
+    public class CourseControllerTests
+    {
+        private Mock<ISearchAndCompareApi> mockApi;
+        private CourseController sut;
+
+        [SetUp]
+        public void SetUp()
+        {
+            mockApi = new Mock<ISearchAndCompareApi>();
+            sut = new CourseController(mockApi.Object);
```

Comment:
`sut` -  I would call this what it is: `courseController` to avoid confusion.

Minor - I wouldn't reject the PR for it but I **really** don't like it.